kata-deploy/action/test-kata.sh

```diff
@@ -83,7 +81,7 @@ function run_test() {
 
       # test pod connectivity:
       kubectl run $busybox_pod --restart=Never --image="$busybox_image" -- wget --timeout=5 "$deployment"
-      waitForProcess "$wait_time" "$sleep_time" "$cmd"
```

Comment:
delete L68 - https://github.com/kata-containers/packaging/pull/844/files#diff-46c7d78a22462bf0ce0158ea0b8d2640R68 as you are not using it?

Reply:
Ouch. Yes.